core/models.py

```diff
@@ -4,3 +4,43 @@
 
 class User(AbstractUser):
     pass
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=26)
```

Comment:
oh, I didnt mean to set it to absolute minimum. But like if max real value is 26, lets to sth like 50 or if you like binary, 64. 26 is way too close to real limit, always leave some spare bytes ;)